dist/livereload.js

```diff
@@ -385,19 +385,31 @@
     LiveReload.prototype.performReload = function(message) {
       var _ref, _ref1;
       this.log("LiveReload received reload request: " + (JSON.stringify(message, null, 2)));
-      return this.reloader.reload(message.path, {
+      this.reloader.reload(message.path, {
         liveCSS: (_ref = message.liveCSS) != null ? _ref : true,
         liveImg: (_ref1 = message.liveImg) != null ? _ref1 : true,
         originalPath: message.originalPath || '',
         overrideURL: message.overrideURL || '',
-        serverURL: "http://" + this.options.host + ":" + this.options.port
+        serverURL: "//" + this.options.host + ":" + this.options.port
```

Comment:
This is actually an outdated build. I since undid that change in CoffeeScript file because I didn't want to introduce unrelated changes in this PR.